### ISSUE TITLE
Powerpoint Online - Slide is not updated after adding a text shape

### ISSUE THREAD

**Issue by theboyknowsclass** [OfficeDev/office-js#2922]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

Powerpoint Online - Slide is not updated after adding a text shape

Using the sample - "Basic API call (TypeScript)" - Run and click the button.

##Expected result:
Shape with textbox is added

##Actual result:
No shape added, 
Preview panel on left hand side is updated though, and if you refresh the browser, the shape appears.

## Your Environment
Powerpoint online
Microsoft Edge
Version 106.0.1370.52 (Official build) (64-bit)
Windows

**Comment by RuizhiSunMS** [comment]
@ononder Please take a look.

**Comment by JeffCharron** [comment]
I have also just ran into this issue. I have found that:

- Resizing the application window will get the shape to appear in the document.
- Manually modifying the slide will get the shape to appear in the document.

My current workaround, which is not ideal, is to:

1. Add my desired shape (issue occurs)
2. Add some placeholder text to the document using `Office.context.document.setSelectedDataAsync()`, which gets my desired shape to appear
3. Immediately delete the placeholder text.
